perception_dataset/rosbag2/rosbag2_to_non_annotated_t4_converter.py

```diff
@@ -899,11 +921,32 @@ def _generate_calibrated_sensor(
                     camera_intrinsic=camera_intrinsic,
                     camera_distortion=camera_distortion,
                 )
+                return calibrated_sensor_token, camera_info
             else:
                 raise ValueError(f"Unexpected sensor modality: {modality}")
 
         return calibrated_sensor_token
```

Comment:
IMHO, now that the modified code assumes to return values in l.924 with `return calibrated_sensor_token, camera_info`, we should do the return for LiDAR and radars within the `if modality in (SENSOR_MODALITY_ENUM.LIDAR.value, SENSOR_MODALITY_ENUM.RADAR.value):` block as well, instead of doing it in l.928, for better consistency. What do you think?

Reply:
How about this https://github.com/tier4/tier4_perception_dataset/pull/156/commits/0dec4a77a0bd75f2e8a279086692b5e44f0b6be6